hknweb/tours/views.py

```diff
@@ -0,0 +1,54 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from django.views import generic
+from django.core.mail import EmailMultiAlternatives
+from django.core.mail import EmailMessage
+from django.template.loader import render_to_string
+from django.views.generic.edit import FormView
```

Comment:
As lgtm mentions, there's some unused imports here. Generally just good to not include those if you don't need them, but they aren't a problem for the functionality here, just can make things slower and make the code a bit harder to read/maintain.